common/app/views/emailEmbed.scala.html

```diff
@@ -0,0 +1,30 @@
+@(metaData: model.Page)(body: Html)(implicit request: RequestHeader)
+
+@import conf.switches.Switches._
+@import common.InlineJs
+@import templates.inlineJS.blocking.js._
+
+<!doctype html>
+<head>
+    @if(play.Play.isDev() || !InlineCriticalCss.isSwitchedOn) {
```

Comment:
hmm, I didn't write this, this file was just renamed from `mainEmail.scala.html` from the earlier email work - don't think it's anything that'd be processed by the inliner.